kyaml/kio/filters/fmtr.go

```diff
@@ -105,11 +117,34 @@ func (f *formatter) fmtNode(n *yaml.Node, path string) error {
 		}
 	}
 	for i := range n.Content {
+		// process the node if configured
+		if f.process != nil {
+			isKey := n.Kind == yaml.MappingNode && i%2 == 0
```

Comment:
I see the comment explaining i%2 in the method below but it might be worth referencing that comment from here for better standalone readability of this method as this part is a bit confusing as to how content list is represented.

Reply:
done